components/StandardDialog.qml

```diff
@@ -38,8 +38,11 @@ import "effects/" as MoneroEffects
 
 Rectangle {
     id: root
-    color: "transparent"
+    color: MoneroComponents.Style.blackTheme ? "black" : "white"
     visible: false
+    radius: 10
+    border.color: MoneroComponents.Style.blackTheme ? Qt.rgba(255, 255, 255, 0.25) : Qt.rgba(0, 0, 0, 0.25)
```

Comment:
Not sure we should use transparency here (there is `appWindowBorderColor`, `grey`, etc.)

Reply:
That’s what I did in 5265e52

Reply:
Changed to `MoneroComponents.Style.moneroGrey` and `MoneroComponents.Style._w_appWindowBorderColor`, which are the same colors used in Figma design schema.